test/library/no-execution-summary-options.test.js

```diff
@@ -0,0 +1,66 @@
+var fs = require('fs'),
+
+    _ = require('lodash'),
+    sh = require('shelljs');
+
+describe('JSON reporter', function () {
+    var outDir = 'out',
+        outFile = outDir + '/newman-report.json';
+
+    beforeEach(function () {
+        sh.test('-d', outDir) && sh.rm('-rf', outDir);
+        sh.mkdir('-p', outDir);
+    });
+
+    afterEach(function () {
+        sh.rm('-rf', outDir);
+    });
+
+    it('should correctly generate the json report with skip-executions option enabled', function (done) {
```

Comment:
I don't think we're testing JSON reporter here.

```suggestion
describe('executionSummary option', function () {
    var outDir = 'out',
        outFile = outDir + '/newman-report.json';

    beforeEach(function () {
        sh.test('-d', outDir) && sh.rm('-rf', outDir);
        sh.mkdir('-p', outDir);
    });

    afterEach(function () {
        sh.rm('-rf', outDir);
    });

    it('when set should add executions in the result', function (done) {
```